### ISSUE TITLE
Taming Feral Wolf or Coyote causes Server to crash 1.15.2

### ISSUE THREAD

**Issue by SayulilyonPipeDream** [itsmeow/betteranimalsplus#174]
**Describe what you were doing before the crash**
Wearing the Hirschgeist Head and trying to feed Feral Wolves' bones will cause a server crash.
 
**To Reproduce**
Steps to reproduce the behavior (if possible):
1. Spawn Feral Wolf
2. Wear Hirschgeist Head
3. Right Click on Feral Wolves with bones
4. Enjoy the crash

**Crash Log (REQUIRED)**
[crash-2020-06-04_22.40.53-server.txt](https://github.com/itsmeow/betteranimalsplus/files/4732610/crash-2020-06-04_22.40.53-server.txt)

**Versions:**
 - MC Version: 1.15.2
 - Mod Version: 8.2.1.96

**Additional context**
I love your mod!

**Comment by itsmeow** [comment]
That is quite an odd error... It seems as though the "tame particle" method ... doesn't exist?? Do you have other mods in your pack that mess with wolves/taming maybe?

**Comment by SayulilyonPipeDream** [comment]
Not that I'm aware of. 

We have Quark, but I believe that changes how they look.

We have Wyrmroost, which adds Dragons available for taming.

Here is the list of mods we are running (Under Related Projects):
https://www.curseforge.com/minecraft/modpacks/modded-minecraft-madames/files/2969114

**Comment by dmliscinsky** [comment]
I experienced a similar issue when trying to tame a Lammergeier by right clicking on it with a (stack of) bones. The server immediately crashed upon me doing so.

**Comment by RadGH** [comment]
I just experienced this error when my girlfriend attempted to tame a Coyote using cooked chicken. It's probably the same, but not sure. Thought I would add to the report. Our crash log:

[crash-2020-06-23_07.30.34-server.txt](https://github.com/itsmeow/betteranimalsplus/files/4817506/crash-2020-06-23_07.30.34-server.txt)

```
java.lang.NoSuchMethodError: its_meow.betteranimalsplus.common.entity.EntityCoyote.func_70908_e(Z)V
	at its_meow.betteranimalsplus.common.entity.EntityCoyote.func_184645_a(EntityCoyote.java:181) ~[?:8.2.1.96] {re:classloading}
	at net.minecraft.entity.MobEntity.func_184230_a(MobEntity.java:953) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A}
...
```

Also it looks like the coyote did eat the cooked chicken after restarting the server but isn't tame.

**Comment by itsmeow** [comment]
Turns out playTameEffect is a client-sided method now. Fixed :)

**Comment by ThePoshTux** [comment]
Sorry to bump, but is this issue meant to fixed for better animals 1.15.2-8.2.1.96 and forge 31.2.36? or is this fix only guaranteed for the forge target of 31.2.24

**Comment by itsmeow** [comment]
It's fixed in the beta builds, no build on CurseForge has this fixed
